protocol-designer/src/load-file/migration/8_5_0.ts

```diff
@@ -161,6 +161,21 @@ export const migrateFile = (
           liquidClassesSupported,
           ...rest
         } = form
+        const tipRackDef = labwareDefinitions[form.tipRack]
+        const pipetteName =
+          equipmentLoadInfoFromCommands.pipettes?.[form.pipette]?.pipetteName ??
+          null
+        const pipetteSpecs =
+          pipetteName != null ? getPipetteSpecsV2(pipetteName) : null
+        const defaultPushOutVolume =
+          pipetteSpecs == null
```

Comment:
nit
const pipetteSpecs: PipetteV2Specs | null

```suggestion
          pipetteSpecs === null
```